microapidev/stockTicker-api
# Add test, and setup Travis CI

**Before submitting your PR for review**

- Run `npm run lint` to find errors in code syntax/format
- Run `npm run lint:fix` to fix all auto-fixable errors in source code and auto-format with prettier
- Ensure you fix any linting errors displayed after running any of the above commands

**What does this PR do?**

Ensures Travis CI builds with tests passing

**description of Task to be completed?**

- Add demo test for server
- Ensure test passes and build sticker is green

**How should this be manually tested?**

- Follow steps in README to clone and setup code in your local machine
- Run `npm run test` to see test pass

**Any background context you want to provide?**

This is to setup  the basis for writing cleaner and healthier code

**What is the link to the issue on Github?**

No issue was created on it

**Questions:**

If something is unclear or you want some questions to be addressed by your peers, mention them here
